lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -610,6 +619,19 @@ def create_transaction_parameters(money, credit_card_or_vault_id, options)
 
         parameters
       end
+
+      def format_card_brand(card_brand)
```

Comment:
Speaking with Jason we agreed it would be best to normalize the card brands with what is used elsewhere in the AM code base